src/contents/articles/hello-world/index.md

```diff
@@ -27,4 +25,4 @@ We want to grow as software developers but we also want to grow as a people. As
 
 ## Join us!
 
-We got a whole lot of things planned out for you, and we really believe we'll all feel better about our profession and passion if we do it together. With namespace, local developers and enthusiasts will finally have that common ground - one module, no globals.
```

Comment:
I would put such details on how members would feel better in the 'Why do we need a community?' subsection, the conclusion shouldn rarely introduce new ideas - didn't intend for this article to be an exception to that